tests/test_mod.py

```diff
@@ -202,26 +202,25 @@ def test_large_query():
 @pytest.mark.scihub
```

Comment:
This test will only fail if cassettes are disabled or newly recorded. Which also means that it won't fail after configuration changes at the Copernicus server side. For the sake of speed of the tests I would still keep the vcr decorator but maybe add a line to the test documentation or testing instructions at the Readme or #119 that `--vcr record_new` should be performed on a semi-regular basis to catch such gremlins.